src/Arcus.Scripting.Sql/Scripts/Invoke-AzSqlDatabaseMigration.ps1

```diff
@@ -0,0 +1,154 @@
+param(
+    [parameter(Mandatory=$true)][string] $ServerName = $(throw "Please provide the name of the SQL Server that hosts the SQL Database. (Do not include 'database.windows.net'"),
+    [parameter(Mandatory=$true)][string] $DatabaseName = $(throw "Please provide the name of the SQL Database"),
+    [parameter(Mandatory=$true)][string] $UserName = $(throw "Please provide the UserName of the SQL Database"),
+    [parameter(Mandatory=$true)][string] $Password = $(throw "Please provide the Password of the SQL Database"),
+    [parameter(Mandatory=$false)][string] $ScriptsFolder = "$PSScriptRoot/sqlScripts",
+    [parameter(Mandatory=$false)][string] $ScriptsFileFilter = ".sql",
```

Comment:
Same here, I think this should be *.sql instead of .sql

Reply:
Confirming - after a test - this should be `*.sql`, as the script didn't detect any files when using `.sql`

Reply:
Ok, let's change that!